README.md

```diff
@@ -231,7 +231,7 @@ http.createServer(async (req, res) => {
   // ...
 
   // Creates a headless Playwright MCP server with SSE transport
-  const connection = await createConnection({ headless: true });
+  const connection = await createConnection({ browser: { launchOptions: { headless: true } } });
```

Comment:
This is supposed to be a SSE transport so I start wondering if we should adopt something similar here in this example like we have in the tests: `sessions` map and GET/POST handling, otherwise it didn't work for me.